sdk/data/azcosmos/cosmos_query_response.go

```diff
@@ -130,3 +130,10 @@ func newDatabasesQueryResponse(resp *http.Response) (QueryDatabasesResponse, err
 type queryDatabasesServiceResponse struct {
 	Databases []DatabaseProperties `json:"Databases,omitempty"`
 }
+
+// ReadManyItemsResponse contains the response from the ReadManyItems operation..
+type ReadManyItemsResponse struct {
+	RequestCharge float32
```

Comment:
Doc comment on this? Something like:


```suggestion
    // The total cost of the operation in RUs
	RequestCharge float32
```

Reply:
added doc comment